### PR TITLE
[vara] 3 sec per block

### PR DESCRIPTION
**Pull request by ukint-vs**
Resolves # .

-
-
-

@gear-tech/dev

### CODE REVIEW

File: runtime/vara/src/lib.rs [hunk 1]
```diff
@@ -433,12 +433,12 @@ impl pallet_gear_staking_rewards::Config for Runtime {
 
 // TODO: review staking parameters - currently copying Kusama
 parameter_types! {
-    // Six sessions in an era (4 hours)
+    // Six sessions in an era (12 hours)
     pub const SessionsPerEra: sp_staking::SessionIndex = 6;
     // 42 eras for unbonding (7 days)
```
Comment (shamilsan) [inline runtime/vara/src/lib.rs:438]: Should we fix this comment too?

Comment (ukint-vs) [reply runtime/vara/src/lib.rs:438]: @ekovalev

Comment (ekovalev) [reply runtime/vara/src/lib.rs:438]: `// 42 eras for unbonding (7 days) ` is obviously misleading given 12 hours epoch duration.
I guess @shamilsan is referring to this comment

**Comment by ekovalev** [comment]
How about the vesting schedules in the chain spec?
We calculate 1 year period after which vesting kicks in based on the old block length.
@NikVolf , will you generate a new spec accordingly?

File: runtime/vara/src/lib.rs (hunk 1 above)
Comment (shamilsan) [reply runtime/vara/src/lib.rs:438]: > `// 42 eras for unbonding (7 days) ` is obviously misleading given 12 hours era duration. I guess @shamilsan is referring to this comment

Exactly.

**Review by NikVolf — APPROVED** [review]

**Pull request merged**